package.json

```diff
@@ -24,17 +24,17 @@
   "author": "Twilio Developer Education",
   "license": "MIT",
   "engines": {
-    "node": ">=8.x"
+    "node": "^12"
```

Comment:
I think since v10 is still LTS it's fair to make this compatible for 10

Reply:
Hi @dkundel. Github Actions are running for both versions 10 and 12. What do you think if we keep the 12 version in the package file?

Reply:
Right now this number means that it has to be node 12 or newer and excludes 10.

Reply:
`>=10.x` would probably be the better one here

Reply:
@dkundel, thanks for your reply. I've changed the node version.